crates/holochain_persistence_lmdb/src/common.rs

```diff
@@ -1,19 +1,21 @@
-use holochain_logging::prelude::*;
-use lmdb::Error as LmdbError;
+//use holochain_logging::prelude::*;
+//use lmdb::Error as LmdbError;
 use rkv::{
     DatabaseFlags, EnvironmentFlags, Manager, Rkv, SingleStore, StoreError, StoreOptions, Value,
+    Writer,
 };
 use std::{
+    collections::HashMap,
     path::Path,
     sync::{Arc, RwLock},
 };
 
 const DEFAULT_INITIAL_MAP_BYTES: usize = 100 * 1024 * 1024;
 
 #[derive(Clone)]
-pub(crate) struct LmdbInstance {
+pub struct LmdbInstance {
```

Comment:
yeahhh does this really need to be `Clone`? If not can we take it out of the RwLock?

Reply:
Can't eliminate the rwlock over `rkv::Rkv`- it's part of the rkv api itself.

Reply:
It's always been clone, I'm taking advantage of it for now. In the future we could try to get rid of that requirement, but at least we can easily send cursors/cas/eavs over channels if we need to.